comms/lighthouse/package.json

```diff
@@ -12,7 +12,6 @@
   },
   "dependencies": {
     "@catalyst/commons": "0.1.0",
```

Comment:
Isn't this dependency deprecated in favour of `@dcl/catalyst-node-commons` ?
Also `@dcl/catalyst-contracts` shouldn't be added here?

I think same is happening in the other `package.json`s, you can run `yarn cleanup && yarn` to re-install dependencies

Reply:
Sorry @guidota, I forgot to add a commit after the rebase, I fixed that and then a couple of extra issues. Thank you very much for your review and for the catch